plugins/vpp/srplugin/vppcalls/srv6.go

```diff
@@ -380,23 +387,31 @@ func (calls *srv6Calls) modPolicy(operation uint8, bindingSid net.IP, policy *sr
 	return nil
 }
 
-func (calls *srv6Calls) convertNextSidList(nextSidList []string) (uint8, []sr.SrIP6Address, error) {
-	segments := make([]sr.SrIP6Address, 0)
-	for _, sid := range nextSidList {
+func (calls *srv6Calls) convertPolicySegment(policySegment *srv6.PolicySegment) (*sr.Srv6SidList, error) {
+	segments := make([]sr.Srv6Sid, 0)
```

Comment:
This doesn't need to be so verbose if you set length to zero, `var segments []sr.Srv6Sid` is enough.

Reply:
fixed in 1f9e360